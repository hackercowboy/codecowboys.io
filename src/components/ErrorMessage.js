import React from 'react';
import PropTypes from 'prop-types';

import { ErrorMessage as FormikErrorMessage } from 'formik';

function ErrorMessage({ name }) {
  return (
    <div className="error-message">
      <FormikErrorMessage name={name} />
    </div>
  );
}

ErrorMessage.propTypes = {
  name: PropTypes.string.isRequired,
};

export default ErrorMessage;
