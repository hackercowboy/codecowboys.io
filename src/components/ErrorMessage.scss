@import '../styles/colors';

.error-message {
  color: $error-color;
  font-size: 0.9em;
}
