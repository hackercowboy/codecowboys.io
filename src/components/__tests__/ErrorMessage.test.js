import React from 'react';
import { shallow } from 'enzyme';

import ErrorMessage from '../ErrorMessage';

describe('<ErrorMessage/>', () => {
  it('should render', () => {
    const wrapper = shallow(<ErrorMessage name="test" />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
