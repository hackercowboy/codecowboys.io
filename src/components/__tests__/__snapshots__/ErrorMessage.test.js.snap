// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage/> should render 1`] = `
"<div className=\\"error-message\\">
  <FormikConnect(ErrorMessageImpl) name=\\"test\\" />
</div>"
`;
